packages/node/src/node.ts

```diff
@@ -1 +1,113 @@
-export default class Node {}
+import * as cf from "@counterfactual/cf.js";
+import { EventEmitter } from "events";
+import * as _ from "lodash";
```

Comment:
leaning towards getting rid of lodash, would recommend implementing this file without it

Reply:
When was this decision made / what's the rationale? My understanding was that from a while ago we decided to converge on using lodash

Reply:
Yeah we should be more explicit about it but I did some thinking and looking around and general consensus right now is that for 90% of the ways people (including us) use lodash, ES6 does that for you. e.g., https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore